R/pkg/R/mllib_regression.R

```diff
@@ -53,12 +53,19 @@ setClass("IsotonicRegressionModel", representation(jobj = "jobj"))
 #'               the result of a call to a family function. Refer R family at
 #'               \url{https://stat.ethz.ch/R-manual/R-devel/library/stats/html/family.html}.
 #'               Currently these families are supported: \code{binomial}, \code{gaussian},
-#'               \code{Gamma}, and \code{poisson}.
+#'               \code{Gamma}, \code{poisson} and \code{"tweedie"}.
+#'               The tweedie family must be specified using character string \code{"tweedie"}. The family
+#'               function \code{tweedie} in the \code{statmod} package is not supported.
```

Comment:
I prefer not having hard dependency on an external package in R, but if statmod is present, could we support `family = tweedie` (not string) too?